styles.css

```diff
@@ -13,11 +13,23 @@
     width: 1em;
     height: 1em;
     vertical-align: middle;
-    margin-left: 0.3em;
+    margin-left: 0.5em;

```

Comment:
What was the thinking behind this change please?
(asking as a non-CSS developer)

Reply:
I thought the space between the last brace and the pencil was a bit small. It should be at bit larger than the size of a space char that separates other elements to make it stand out as a separate element, particularly as the pencil is actually not part of the task.

Reply:
> I thought the space between the last brace and the pencil was a bit small. It should be at bit larger than the size of a space char that separates other elements to make it stand out as a separate element, particularly as the pencil is actually not part of the task.

Nice. 

I would have liked that to be mentioned in the machine-generated changelog for the next release. It's only a tiny change, but it's a nice improvement, and someone may care to know about the diffs.

By it being inside this PR, it won't get mentioned in the release notes, and when I merge this PR, I will have to remember to add an extra bullet point mentioning the extra change, when editing the commit message.

I'll change the summer for this commit, for now.